features/expectation_framework_integration/configure_expectation_framework.feature

```diff
@@ -72,7 +72,6 @@ Feature: configure expectation framework
       """
     And  the output should contain "3 examples, 1 failure"
 
-  @broken-on-jruby-9000
```

Comment:
Surely this needs reverting, as all JRuby is 9000? Or is it fixed in 9.1/9.2?

Reply:
It passes just fine. Was apparently broken on 9.0 only.